feature-tests/run-tests.sh

```diff
@@ -0,0 +1,27 @@
+#! /bin/bash
+set -u
+
+# in pipeline copy files to CODEBUILD_SRC_DIR where next to the .env file is created
+cp /package.json . 2>/dev/null || :
+cp /tsconfig.json . 2>/dev/null || :
+cp /jest.config.ts . 2>/dev/null || :
+cp /jest-cucumber-config.js . 2>/dev/null || :
+cp -R /tests ./tests 2>/dev/null || :
+cp -R /apiEndpoints ./apiEndpoints 2>/dev/null || :
+cp -R /node_modules ./node_modules 2>/dev/null || :
+cp -R /utils ./utils 2>/dev/null || :
+
+# run tests and save the exit code
+declare test_run_result
+export tagFilter=@regression
```

Comment:
Can we set this from a parameter so we can change it in different environments
  export tagFilter =$(aws ssm get-parameter --name "/tests/${STACK_NAME}/TestTag" | jq -r ".Parameter.Value")

Reply:
Export filter for Test Tag update on run-tests.sh file